### ISSUE TITLE
problematic args description table of aixs in tf.nn.softmax

### ISSUE THREAD

**Issue by Yonma** [tensorflow/tensorflow#46819]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/nn/softmax#args

document version: 2.4.1

## Description of issue (what needs changing):

In the args description table, it says axis default to -1,
while the code below 'view alias' uses: axis=None.

and according to its equivalent code, axis deafaults to None in the function reduce_sum.

**Comment by around-star** [comment]
@Yonma  I think the default value of the axis param is -1 .

**Comment by around-star** [comment]
@Yonma  The value of the param axis should be -1.

**Comment by ymodak** [comment]
The arg description is correct. `Axis=-1` by default.
https://github.com/tensorflow/tensorflow/blob/85c8b2a817f95a3e979ecd1ed95bff1dc1335cff/tensorflow/python/ops/nn_ops.pySee #L3732-L3733

**Comment by Yonma** [comment]
I got it! Thank u very much, I forgot to check such thing in the sourcecode, but its really confusing at the first sight.
